source/core/utilities.cc

```diff
@@ -361,6 +361,17 @@ create_output_folder(const std::string &dirname)
 #endif
 }
 
+void
+delete_output_folder(const std::string &dirname)
+{
+#if __GNUC__ > 7
+  std::filesystem::remove_all(dirname);
+#else
+  // std::string command = "rm -rf " + dirname;
+  // system(command.c_str());
+#endif
+}
+
```

Comment:
Good addition, but where is it used? Or how is it used :)

Reply:
In "set_initial_conditions()" of `vof_algebraic_interface_reinitialization.cc`. Where I first check if reinitialization steps should be outputted or not. This is done in order to avoid that files from previous reinitialization processes remain in the output folder.

It could also be used for our simulation outputs (not done in this PR) so that everytime we rerun a simulation (not a restart), we scrap previous pvtu other outputted files.